### ISSUE TITLE
[Feature request] dark mode

### ISSUE THREAD

**Issue by harshhhdev** [codeforcauseorg/codeforcause.org#144]
Add a simple setting which makes the website go light, or dark, for the creatures of the night.

**Comment by Abhishek-kumar09** [comment]
Fantastic one!
We thought of it earlier but could not apply due to several other things on pipeline.
Do you want to got for it?

**Comment by harshhhdev** [comment]
Yeah, glad to do it!

**Comment by Abhishek-kumar09** [comment]
Great! Thanks for showing your interest :)

**Comment by chinmaym07** [comment]
Hey @Abhishek-kumar09 can I work upon this issue .?

**Comment by harshhhdev** [comment]
I really apologise, with school and other side projects I haven't found the time to really work on this. Hope you can understand.

**Comment by Abhishek-kumar09** [comment]
No worries, I understand. But I am re-opening this issue, so that other could come across and solve it. Thank You for opening it.

**Comment by sarvesha-jaiswal** [comment]
is the issue is still open ? can i work on it?

**Comment by Abhishek-kumar09** [comment]
Yes, here is the theme written: https://github.com/codeforcauseorg/codeforcause.org/blob/development/src/theme/index.js

You can change all the points were colors were written explicitly to the theme variables

**Comment by sarvesha-jaiswal** [comment]
yes sure!

**Comment by Anishpras** [comment]
Would like to work on this.

**Comment by sarvesha-jaiswal** [comment]
@Anishpras sorry, but i have started working on this issue, maybe if i need any help i will contact you

Thanks

**Comment by khushbooag4** [comment]
Is this issue still open

**Comment by Abhishek-kumar09** [comment]
Oh hi @khushbooag4 Sorry I was missing the notifications for the vacations.  I think this issue isn't fixed. 
Better to able to create a toggle for dark mode. 
The theme is already defined for dark:

https://github.com/codeforcauseorg/codeforcause.org/blob/0a5308d6772edb47508b4d012314c13e7fe941e5/src/theme/index.js#L69-L98

and applying a dark theme is done in mui in an easy way: https://adityanaik.dev/blog/2020/05/20/material-ui-how-to-easily-set-up-dark-theme-toggle-in-react/

@KeenWarrior @GangaChatrvedi will let you know more about the need of the dark mode in the current UI architecture

**Comment by KeenWarrior** [comment]
@khushbooag4 We do not have the figma sheets ready for what we expect in dark mode. So completion of this issue will become ambiguous at this moment.
